File: benchmarks/bench_0.go
package main

import (
	"fmt"
	"time"
)

func fib(n int64) int64 {
    if n < 2 {
        return n
    }
    return fib(n-1) + fib(n-2)
}

func main() {
    t1 := time.Now()
    res := fib(35)
    t2 := time.Now()
    fmt.Println(res)
    fmt.Printf("%.6f\n", t2.Sub(t1).Seconds())
}

File: benchmarks/bench_0.py
import time
import sys
sys.setrecursionlimit(200000)

def fib(n):
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)


t1 = time.perf_counter()
res = fib(35)
t2 = time.perf_counter()
print(res)
print(f"{t2 - t1:.6f}")
